apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-responsive-padding-example.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+    selector: 'fdp-dynamic-page-responsive-padding-example',
+    templateUrl: './platform-dynamic-page-responsive-padding-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformDynamicPageResponsivePaddingExampleComponent {
+    onCollapseChange(event: Event): any {
```

Comment:
Event type "any" needs to replaced with specfic event types such as Keyboard, mouse etc, same as other components has done.

Reply:
added appropriate event type and return type as void.